doc/controllers_index.rst

```diff
@@ -0,0 +1,16 @@
+.. _controllers:
+
+Controllers user documentation
+==============================
+
+.. include:: ../diff_drive_controller/doc/userdoc.rst
```

Comment:
I would not include them all into one file, but rather reference them. Why not simply using `index.rst` as the main file in the each directory and than simply auto-generate all the links to them:

See here:
https://github.com/StoglRobotics/ros_team_workspace/blob/rolling/docs/use-cases/index.rst

This generate nice menu structure in the docs (check [here](https://stoglrobotics.github.io/ros_team_workspace/use-cases/index.html))

Reply:
It looks pretty much the same to me:

![Screenshot from 2021-03-17 07-18-04](https://user-images.githubusercontent.com/3524577/111429529-02f84f00-86f1-11eb-8524-ded900179b2e.png)

Happy to have an alternative approach but I don't wish to spend more time on this now.